frontend/src/homepage/DevicesCard.jsx

```diff
@@ -8,10 +8,14 @@ export default function DevicesCard() {
   return (
     <DashboardCard title="Devices">
       <Grid item xs={4}>
-        <DashBoardButton href="/Devices">View Senders</DashBoardButton>
+        <DashBoardButton href="/Devices" selected={1}>
```

Comment:
I think you should rename the import as `DashboardButton` instead of `DashBoardButton`

Reply:
82813b9